adoc/deployment-bootstrap.adoc

```diff
@@ -798,3 +798,32 @@ kube-system   kured-zv696                             1/1       Running   0
 kube-system   oidc-dex-55fc689dc-b9bxw                1/1       Running   0          2m
 kube-system   oidc-gangway-7b7fbbdbdf-ll6l8           1/1       Running   0          2m
 ----
+
+==== Co-installing multiple versions of kubectl
+
+It's possible to install multiple versions of `kubectl` on the same machine. In
+a {productname} management machine, you will be able to co-install multiple
+versions of `kubectl`, and later mark which is the version to be used with
+`update-alternatives`.
+
+This can be useful, for example, when managing multiple Kubernetes clusters from
+the same {productname} management machine. That is, following {productname} 4.5,
+{productname} will be able to manage other (supported) Kubernetes versions than
+only 1.17.
+
+For example, in a system like this:
+
+[source,bash]
+----
```

Comment:
@r0ckarong I don't like that this code block does not have horizontal scrolling. Is there any way in AsciiDoc to force this?

Reply:
The final output will have horizontal scrolling. This depends on the settings of the generator processing the input files. What are you using to preview this?